browser/components/MarkdownEditor.js

```diff
@@ -213,6 +213,8 @@ class MarkdownEditor extends React.Component {
     if (!(editorFontSize > 0 && editorFontSize < 101)) editorFontSize = 14
     let editorIndentSize = parseInt(config.editor.indentSize, 10)
     if (!(editorFontSize > 0 && editorFontSize < 132)) editorIndentSize = 4
+    let editorLineNumber = config.editor.lineNumber
+    if (editorLineNumber === undefined) editorLineNumber = true
```

Comment:
`const editorLineNumber = config.editor.lineNumber || true` or something?

Reply:
If `config.editor.lineNumber` were `false` (disabled), `config.editor.lineNumber || true` would be `true`. I don't think that's what you want.

Reply:
What are cases that `config.editor.lineNumber` to be `undefined`? The setting should be `true` or `false`.

Reply:
Oh, sorry, of course, you are right. I used that before I declared the property in the `ConfigManager.js`.

Reply:
> If config.editor.lineNumber were false (disabled), config.editor.lineNumber || true would be true. I don't think that's what you want.

I got it.

Reply:
I'm confused, sorry. The variable can't be undefined (as you said), can it?

Reply:
Oh, I was confused, too lol

Forget about the ternary operation.

Reply:
Or we can just use `config.editor.lineNumber` directly [here](https://github.com/xxdavid/Boostnote/blob/7a4258bb208d43be80e575fe61c0a0a187811e3f/browser/components/MarkdownEditor.js#L247), I think 😄 .